indico/modules/events/editing/client/js/editing/page_layout/MenuBar.jsx

```diff
@@ -6,58 +6,85 @@
 
-export default function MenuBar({eventId, menuItems}) {
+function EditableListMenu({eventId, type}) {
+  if (location.pathname === editableTypeListURL({confId: eventId, type})) {
+    return (
+      <Menu vertical>
+        <Menu.Item active>
+          <Translate>Editable list</Translate>
+        </Menu.Item>
+      </Menu>
+    );
+  }
+  return (
+    <Menu vertical>
+      <Menu.Item>
+        <Link to={editableTypeListURL({confId: eventId, type})}>
+          <Translate>Editable list</Translate>
+        </Link>
+      </Menu.Item>
+    </Menu>
+  );
+}
+
+EditableListMenu.propTypes = {
+  eventId: PropTypes.number.isRequired,
+  type: PropTypes.string.isRequired,
+};
+
+export default function MenuBar({eventId, menuItems, type}) {
   return (
     <div styleName="menu-bar">
       <Header as="h2" styleName="header">
-        <Translate>Paper Editing</Translate>
+        {EditableEditingTitles[type]}
       </Header>
-      <ul styleName="list">
-        <li>
-          <span styleName="capitalized">
+      <EditableListMenu eventId={eventId} type={type} />
+      <Menu vertical>
+        <Menu.Item header>
+          <span styleName="capitalized" style={{color: Palette.black}}>
             <Translate>other modules</Translate>
           </span>
-          <ul styleName="inner-list">
-            {menuItems.map(item => (
-              <a key={item.name} href={item.url}>
-                <li styleName="inner-list-item">
-                  {item.icon && <Icon name={item.icon.replace('icon-', '')} />}
-                  {item.title}
-                </li>
-              </a>
-            ))}
-          </ul>
-        </li>
-      </ul>
-      <ul styleName="list">
-        <li>
-          <span styleName="capitalized">
+        </Menu.Item>
+        {menuItems.map(item => (
+          <Menu.Item key={item.name} name={item.name}>
+            <a href={item.url}>
+              {item.icon && <Icon name={item.icon.replace('icon-', '')} />}
+              {item.title}
+            </a>
+          </Menu.Item>
+        ))}
+      </Menu>
+      <Menu vertical>
+        <Menu.Item header>
+          <span styleName="capitalized" style={{color: Palette.black}}>
             <Translate>other views</Translate>
           </span>
-          <ul styleName="inner-list">
-            <a href={displayURL({confId: eventId})}>
-              <li styleName="inner-list-item">
-                <Icon name="tv" />
-                <Translate>Display</Translate>
-              </li>
-            </a>
-            <a href={managementURL({confId: eventId})}>
-              <li styleName="inner-list-item">
-                <Icon name="pencil" />
-                <Translate>Management</Translate>
-              </li>
-            </a>
-          </ul>
-        </li>
-      </ul>
+        </Menu.Item>
+        <Menu.Item name="display">
+          <a href={displayURL({confId: eventId})}>
+            <Icon name="tv" />
+            <Translate>Display</Translate>
```

Comment:
Can you add a space between those two elements (same below)? Right now the text and icon don't have any space between which looks a bit weird. Replacing the linebreak with a space should work (not sure if prettier prefers that or `{' '}` here)